integration_tests/src/main/python/window_function_test.py

```diff
@@ -68,11 +68,44 @@ def test_window_aggs_for_rows(data_gen):
         'from window_agg_table ')
 
 
+_grpkey_longs_with_decimals = [
+    ('a', RepeatSeqGen(LongGen(nullable=False), length=20)),
+    ('b', DecimalGen(precision=18, scale=3, nullable=False)),
+    ('c', IntegerGen())]
+
+_grpkey_longs_with_nullable_decimals = [
+    ('a', RepeatSeqGen(LongGen(nullable=(True, 10.0)), length=20)),
+    ('b', DecimalGen(precision=18, scale=10, nullable=True)),
+    ('c', IntegerGen())]
+
+
+@ignore_order
+@pytest.mark.parametrize('data_gen', [_grpkey_longs_with_decimals,
+                                      _grpkey_longs_with_nullable_decimals], ids=idfn)
+def test_window_aggs_for_rows_on_decimal(data_gen):
+    # TODO: test sum aggregation when underlying cuDF implementation is ready
```

Comment:
nit: If there is a follow on issue for this it would be great to put the issue number here too.

Reply:
Issue link has been added.